infra/base-images/all.sh

```diff
@@ -19,4 +19,4 @@ docker build --pull -t ossfuzz/base $@ infra/base-images/base
 docker build -t ossfuzz/base-clang $@ infra/base-images/base-clang
 docker build -t ossfuzz/base-libfuzzer $@ infra/base-images/base-libfuzzer
 docker build -t ossfuzz/libfuzzer-runner $@ infra/base-images/libfuzzer-runner
-
+docker build -t ossfuzz/coverage $@ infra/base-images/coverage
```

Comment:
You will also need to change Jenkinsfile in this folder & Jenkinsfile in ../push-images/